godtools/App/Services/RealmDatabase/RealmDatabase.swift

```diff
@@ -12,7 +12,7 @@ import RealmSwift
 class RealmDatabase {
     
     private static let config: Realm.Configuration = RealmDatabase.createConfig
-    private static let schemaVersion: UInt64 = 5
+    private static let schemaVersion: UInt64 = 6
```

Comment:
out of curiosity does Realm on iOS automatically generate migrations? or does just bumping the schemaVersion reset the database completely?

Reply:
from: https://www.mongodb.com/docs/realm/sdk/swift/examples/modify-an-object-schema/

> If your schema update adds optional properties or removes properties, Realm Database can perform the migration automatically. You only need to increment the schemaVersion.

That's nice

Reply:
Yeah they do some things automatically.